data/arweave/ecosystem/anothermoney/meta.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Another Money",
+  "description": "Another Coin is the first serious non-PST token based on arweave blockchain. Made specially for usage in SmartWeave Contracts.",
+  "web": "https://another.money",
+  "categories": [
+    "coins"
```

Comment:
any better category to put?

Reply:
Any. It's created as coin for smart contracts. Maybe also add smartcontracts 🤔